SkietbaanBE/SkietbaanBE/Controllers/CompetitionController.cs

```diff
@@ -58,14 +60,20 @@ public async Task<Competition> CompetitionGetByName(string Name)
         [HttpPost]
         public async Task<IActionResult> AddCompetition([FromBody]Competition comp)
         {
-            if (!ModelState.IsValid)
+            if (ModelState.IsValid)
             {
-                return BadRequest(ModelState);
+                Competition dbComp = _context.Competitions.FirstOrDefault(C => C.Name == comp.Name);
```

Comment:
use a lower case letter for these Linq queries

Reply:
Done